schema/nlog.schema.json

```diff
@@ -0,0 +1,559 @@
+                    "minLevel": "Trace",
+                    "writeTo": "MyClass-file"
+                  }
+                ],
+                "required": [
+                  "logger"
+                ],
+                "additionalProperties": true,
+                "properties": {
+                  "logger": {
+                    "$id": "#/NLog/properties/rules/item/properties/logger",
+                    "type": "string",
+                    "title": "logger",
+                    "description": "Filter on the name of the logger. May include wildcard characters ('*' or '?').",
+                    "examples": [
+                      "*",
+                      "Microsoft.*",
+                      "*.Something.*"
+                    ]
+                  },
+                  "ruleName": {
+                    "$id": "#/NLog/properties/rules/item/properties/rulename",
+                    "type": "string",
+                    "title": "rulename",
+                    "description": "Rule identifier to allow rule lookup with Configuration.FindRuleByName and Configuration.RemoveRuleByName.",
+                    "examples": [
+                      "*"
+                    ]
+                  },
+                  "level": {
+                    "$id": "#/NLog/properties/rules/item/properties/level",
+                    "type": "string",
+                    "title": "level",
+                    "description": "",
+                    "enum": [
+                      "Trace",
+                      "Debug",
+                      "Info",
+                      "Warn",
+                      "Error",
+                      "Fatal"
+                    ]
+                  },
+                  "levels": {
+                    "$id": "#/NLog/properties/rules/item/properties/levels",
+                    "type": "string",
+                    "title": "levels",
+                    "description": "Comma separated list of levels that this rule matches.",
+                    "examples": [
+                      "Trace, Debug",
+                      "Warn, Error, Fatal",
+                      "Debug"
+                    ]
+                  },
+                  "minLevel": {
+                    "$id": "#/NLog/properties/rules/item/properties/minLevel",
+                    "type": "string",
+                    "title": "minLevel",
+                    "description": "",
+                    "enum": [
+                      "Trace",
+                      "Debug",
+                      "Info",
+                      "Warn",
+                      "Error",
+                      "Fatal"
+                    ]
+                  },
+                  "maxLevel": {
+                    "$id": "#/NLog/properties/rules/item/properties/maxLevel",
+                    "type": "string",
+                    "title": "maxLevel",
+                    "description": "",
+                    "enum": [
```

Comment:
Will `enum` not restrict the available options? Since you can use layout-renderers now.